src/commands/generate/generators/scaffold/handlers/model.js

```diff
@@ -16,7 +16,18 @@ const get${capSingular} = async function (key) {
 }
 
 const get${capPlural} = async function () {
-  return data.get({ table: '${plural}' })
+  const pages = await data.get({
+    table: '${plural}',
+    limit: 25
```

Comment:
This limit isn't arbitrary as in I landed on this limit through trial and error. AWS is supposed to limit at around 20 but in practice I am seeing it limit at ~15. Bumping the limit to 25 returns expected results. This could be tinkered with more to fine-tune perf if need be.